CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ this is the last Prometheus release that supports macOS 10.12 Sierra.
 * [BUGFIX] Fix timestamp handling in OpenMetrics parser. #9008
 * [BUGFIX] Fix potential duplicate metrics in `/federate` endpoint when specifying multiple matchers. #8885
 * [BUGFIX] Fix server configuration and validation for authentication via client cert. #9123
+* [BUGFIX] Fix start and end keywords masking label and metric names when using PromQL `@` modifier. #9119
```

Comment:
```suggestion
* [BUGFIX] Allow `start` and `end` again as label names in PromQL queries. They were disallowed since the introduction of @ timestamp feature.  #9119
```

Maybe that is more clear?